qtopconf.yaml

```diff
@@ -1,3 +1,54 @@
+schedulers:
+  pbs:
+    pbsnodes_file: pbsnodes_a.txt
+    qstatq_file: qstat_q.txt
+    qstat_file: qstat.txt
+  oar:
+    oarnodes_s_file: oarnodes_s
+    oarnodes_y_file: oarnodes_y
+    oarnodes_file: oarnodes
+    oarstat_file: oarstat
+  sge:
+    sge_file: sge.xml
+
+#scheduler: pbs
+scheduler: oar
+
+# insert comment here
+state_abbrevs:
+  pbs:
+    Q: queued_of_user
+    R: running_of_user
+    C: cancelled_of_user
+    E: exiting_of_user
+    W: waiting_of_user
+  oar:
+    E: Error
+    F: Finishing
+    S: Resuming
+    H: Hold
+    L: waiting_of_user
+    W: queued_of_user
+    R: running_of_user
+    T: exiting_of_user
+    S: cancelled_of_user
+
+# original:
+# state_abbrevs = {
+#     'W': 'Waiting',
+#     'L': 'toLaunch',
+#     'L': 'Launching',
+#     'H': 'Hold',
+#     'R': 'Running',
+#     'T': 'Terminated',
+#     'E': 'Error',
+#     'E': 'toError',
+#     'F': 'Finishing',
+#     'S': 'Suspended',
+#     'S': 'Resuming',
+#     'W': 'toAckReservation',
+# }
+
```

Comment:
then, it should be either a tunable or, start spitting out ideas for a helpfile (.rst?) - if it is really meant as configurable inside yaml, it should be easy to switch to it. (perhaps this is a new issue for a future PR)